### ISSUE TITLE
don't put dynamic data into testcase name

### ISSUE THREAD

**Issue by kparal** [fedora-modularity/check_modulemd#12]
Hello, recently check_modulemd started to report results into resultsdb with testcases like this:

```
dist.modulemd.check_modulemd.ModulemdTest.test_dependencies;run-0404 
dist.modulemd.check_modulemd.ModulemdTest.test_dependencies;run-0672 	
dist.modulemd.check_modulemd.ModulemdTest.test_dependencies;run-09bd 	
dist.modulemd.check_modulemd.ModulemdTest.test_dependencies;run-0b2b 	
dist.modulemd.check_modulemd.ModulemdTest.test_dependencies;run-0c0a 	
dist.modulemd.check_modulemd.ModulemdTest.test_dependencies;run-0f64
```
See more currently at https://taskotron.fedoraproject.org/resultsdb/testcases?page=4 .

Please don't put dynamic data into testcases. They are not meant for that. If you dump thousand or more values into testcases, we will not be able e.g. to autocomplete them in the search UI.

The testcase should be `dist.modulemd.check_modulemd.ModulemdTest.test_dependencies` and anything more you need should be an additional keyval in results.

**Comment by kparal** [comment]
Folks, I need you to respond to this.

**Comment by kparal** [comment]
This has been fixed in #23.
